core/src/test/scala/unit/kafka/utils/QuotaUtilsTest.scala

```diff
@@ -17,24 +17,21 @@
 
 package kafka.utils
 
-import java.util.concurrent.TimeUnit
-
 import org.apache.kafka.common.MetricName
-import org.apache.kafka.common.metrics.{KafkaMetric, MetricConfig, Quota, QuotaViolationException}
 import org.apache.kafka.common.metrics.stats.{Rate, Value}
-
-import scala.jdk.CollectionConverters._
+import org.apache.kafka.common.metrics.{KafkaMetric, MetricConfig, Quota, QuotaViolationException}
 import org.junit.Assert._
 import org.junit.Test
-import org.scalatest.Assertions.assertThrows
+
+import java.util.concurrent.TimeUnit
 
 class QuotaUtilsTest {
 
   private val time = new MockTime
   private val numSamples = 10
   private val sampleWindowSec = 1
   private val maxThrottleTimeMs = 500
-  private val metricName = new MetricName("test-metric", "groupA", "testA", Map.empty.asJava)
+  private val metricName = new MetricName("test-metric", "groupA", "testA", java.util.Collections.emptyMap())
```

Comment:
my bad and will revert it. (IntelliJ complained about the erased types)

Reply:
I would ignore IntelliJ issues if it compiles without warnings. I think there are a few more changes like this in the middle of thousands of lines of diff. Something to avoid in the future. :)

Reply:
Please give me one second to revert those changes :(